src/lighteval/tasks/lighteval_task.py

```diff
@@ -61,6 +61,8 @@
 if TYPE_CHECKING:
     from lighteval.logging.evaluation_tracker import EvaluationTracker
 
+FormatterType = Callable[[dict, str], Doc]
```

Comment:
Why do you include a string? 
I don't see which prompt formater takes a string as input